examples/native/native_test.c

```diff
@@ -1221,16 +1215,9 @@ int TPM2_Native_Test(void* userCtx)
         cmdIn.create.inSensitive.sensitive.userAuth.size);
     cmdIn.create.inPublic.publicArea.type = TPM_ALG_SYMCIPHER;
     cmdIn.create.inPublic.publicArea.nameAlg = TPM_ALG_SHA256;
-#ifdef WOLFTPM_MCHP
-    /* workaround for issue with both sign and decrypt being set */
     cmdIn.create.inPublic.publicArea.objectAttributes = (
         TPMA_OBJECT_sensitiveDataOrigin | TPMA_OBJECT_userWithAuth |
         TPMA_OBJECT_noDA | TPMA_OBJECT_decrypt);
-#else
-    cmdIn.create.inPublic.publicArea.objectAttributes = (
-        TPMA_OBJECT_sensitiveDataOrigin | TPMA_OBJECT_userWithAuth |
-        TPMA_OBJECT_noDA | TPMA_OBJECT_sign | TPMA_OBJECT_decrypt);
```

Comment:
Marking this for further comment in the PR - "Generating a key for both signing/encryption and decryption should be possible."